doc/quickstart/mapserver_quickstart.rst

```diff
@@ -52,16 +53,17 @@ Creating my first Mapfile
 
 #. Open any text editor (e.g. :menuselection:`Applications --> Accessories -->
    FeatherPad`).
-#. Create the file "mapserver_quickstart.map" in your home directory:
-   :file:`/home/user/mapserver_quickstart.map`
+#. Create the file "mapserver_quickstart.map" at the following location:
```

Comment:
The idea was always to discourage users to use the system path and use their home directory

Reply:
@kalxas - the location of the Mapfiles is validated by the regex at https://github.com/OSGeo/OSGeoLive/blob/a566ec01ecd594e7dff5c95a4db46a7af22ca8c8/app-conf/mapserver/mapserver.conf#L3 - this currently limits it to `/usr/local/**/*` where the various apps and demos are located. I can change the regex to allow Mapfiles to be loaded from any location (I don't think this would be much of a risk?), or attempt to write one that allows 2 locations (although the regex is already at the limits of my understanding!).